entries/QUnit.stack.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="QUnit.stack" return="String">
+	<title>QUnit.stack()</title>
+	<signature>
+		<argument name="offset" type="Number" optional="true" default="0">
+			<desc>Set the stackTrace line offset.</desc>
+		</argument>
+	</signature>
+	<desc>Returns a single line string represeting the stacktrace (call stack)</desc>
+	<longdesc>
+		<p>This method returns a single line string representing the stacktrace from where it was called. According to its offset argument, QUnit.stack will return the correspondent line from the call stack.</p>
+		<p>The default offset is 0 and will return the current location where it was called.</p>
+		<p>Some browsers doesn't support stacktrace, and <code>QUnit.stack()</code> will return an empty string on them. See more on <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack">MDN</a>.</p>
```

Comment:
Afaik we return `undefined`: https://github.com/leobalter/qunit/blob/external-stack/src/core.js